### DISCUSSION TITLE
Did I miss something

### DISCUSSION THREAD

**Discussion by rabol** [spatie/laravel-markdown#52] [Q&A]
Hi

I was under the impression that code would be highlighted

I'm just using the sample
`
                        <x-markdown>
                        # My title

                        This is a [link to our website](https://spatie.be)

                        ```php
                        echo 'Hello world';
                        ```
                    </x-markdown>

![Screenshot 2023-04-26 at 21 32 09](https://user-images.githubusercontent.com/1177191/234683232-e150ab2c-1b1a-406e-9856-7ab04e89c3eb.png)

is that the expected result ?

**Comment by parallels999** [comment]
[laravel-markdown/installation](https://spatie.be/docs/laravel-markdown/installation-setup)

**Reply by rabol** [reply]
Yes, but the installation only say how to i stall shiki but not how to get it working 

e.g npm install is not the only thing one needs to do, and the installation instructions  only do the npm install!

**Reply by parallels999** [reply]
- https://github.com/spatie/laravel-markdown/discussions/25

Maybe you have interference with another package

**Comment by jeffreyvr** [comment]
It's probably due to indentation within your blade component @rabol. If you remove any indentation on the markdown, you'll probably see the correct result.

**Reply by damms005** [reply]
@freekmurze I think this should be mentioned in the docs, sire
